css/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,8 @@
+.moon-badge-overlay-support-scrim {
+	display: block;
+	z-index: 1000;
+	width: 100%;
+	height: 55px;
```

Comment:
We made a decision recently to make all measurements a multiple of 3, so they can be converted to rem units "smoothly". The easiest way is to +/- 1 to any number. Although it works out even better if it's a multiple of 6, 12 or even 24.